test/quantization/test_workflow_module.py

```diff
@@ -695,6 +696,37 @@ def test_observer_scriptable(self, qdtype, qscheme):
         loaded = torch.jit.load(buf)
         self.assertTrue(torch.equal(obs.get_tensor_value()[0], loaded.get_tensor_value()[0]))
 
+class TestHistogramObserver(QuantizationTestCase):
```

Comment:
Wondering if the tests in def test_save_load_state_dict_script(self) already cover this test.

Reply:
that test scripted the model but didn't save to file, and this problem happens at save step.